internal/querynode/index_loader.go

```diff
@@ -153,85 +101,6 @@ func (loader *indexLoader) printIndexParams(index []*commonpb.KeyValuePair) {
 	}
 }
 
-func (loader *indexLoader) indexParamsEqual(index1 []*commonpb.KeyValuePair, index2 []*commonpb.KeyValuePair) bool {
-	if len(index1) != len(index2) {
-		return false
-	}
-
-	for i := 0; i < len(index1); i++ {
-		kv1 := *index1[i]
-		kv2 := *index2[i]
-		if kv1.Key != kv2.Key || kv1.Value != kv2.Value {
-			return false
-		}
-	}
-
-	return true
-}
-
-func (loader *indexLoader) fieldsStatsIDs2Key(collectionID UniqueID, fieldID UniqueID) string {
-	return strconv.FormatInt(collectionID, 10) + "/" + strconv.FormatInt(fieldID, 10)
-}
-
-func (loader *indexLoader) fieldsStatsKey2IDs(key string) (UniqueID, UniqueID, error) {
-	ids := strings.Split(key, "/")
-	if len(ids) != 2 {
-		return 0, 0, errors.New("illegal fieldsStatsKey")
-	}
-	collectionID, err := strconv.ParseInt(ids[0], 10, 64)
-	if err != nil {
-		return 0, 0, err
-	}
-	fieldID, err := strconv.ParseInt(ids[1], 10, 64)
-	if err != nil {
-		return 0, 0, err
-	}
-	return collectionID, fieldID, nil
-}
-
-func (loader *indexLoader) updateSegmentIndexStats(segment *Segment) error {
```

Comment:
better not to delete this function, is it not easy to add a single test?

Reply:
This function has been deprecated, query node dosen't need to send index stats to msgstream anymore.

Reply:
I would comment these codes